_includes/homepage/infocols.html

```diff
@@ -0,0 +1,64 @@
+{%- comment -%}
+  include: 
+    subtitle
+    title
+    url
+    linktext
+    infoboxes[]:
+      title
+      description
+      url
+{%- endcomment -%}
+<section class="px-8 px-sm-16 px-md-24 py-24 {{ gray_container_class }}">
+  <div class="infocols-container">
+    <div class="infocols-section is-flex">
+      <div class="infocols-header is-flex">
+        <p class="subtitle-2">
+            {{ include.subtitle }}
+        </p>
```

Comment:
The subtitle should be optional so it should be wrapped inside an if-block

Reply:
good point - done, same for all the other optional fields too